var/spack/repos/builtin/packages/py-prometheus-client/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPrometheusClient(PythonPackage):
+    """Prometheus instrumentation library for Python applications."""
+
+    homepage = "https://github.com/prometheus/client_python"
+    url      = "https://github.com/prometheus/client_python/archive/v0.5.0.tar.gz"
```

Comment:
is there a pypi.io download available? If so that one is preferred. Github archives are not necessarily hash stable